App/Containers/OnboardingScreen/index.tsx

```diff
@@ -72,19 +70,20 @@ class OnboardingScreen extends React.Component<Props, State> {
   }
 
   nextPage = () => {
-    if (this.pages && this.pages.props.children.length - 1 > this.state.currentPage && !this.state.blockNext) {
-      this.setState({blockNext: true})
+    if (this.pages && this.pages.props.children.length - 1 > this.state.currentPage) {
+      this.setState({
+        showArrow: this.showArrowForIndex(this.state.currentPage + 1)
+      })
       this.pages.scrollToPage(this.state.currentPage + 1)
-      setTimeout(() => {
-        this.setState({
-          blockNext: false,
-          showArrow: this.showArrowForIndex(this.state.currentPage + 1),
-          currentPage: this.state.currentPage + 1
-        })
-      }, 350)
```

Comment:
`nextPage` now simply tells the pager component to scroll to the next page and sets the arrow visibility based on what the next page will be.